Assignment_5/Assignment_5_Problem_2.js

```diff
@@ -0,0 +1,16 @@
+//Assignment 5
+// Problem 2
+// Prolem statement 
+// http://www.codewars.com/kata/extract-nested-object-reference
+
+// Solution
+
+
+Object.prototype.hash = function(string) {
+    var properties = string.split(".");
+    var myObj = this;
+    for (let i = 0; i < properties.length; i++) {
+        !!myObj && (myObj = myObj[properties[i]]);
```

Comment:
Explore Boolean(myObj)

Reply:
I read about Boolean(obj) but couldn't get where to implement it here.. i will search and update here..

Reply:
!!myObj can be replaced with Boolean(myObj)